packages/contracts/README.md

```diff
@@ -92,98 +90,58 @@ yarn build:contracts
 ```
 
 ### Deploying the Contracts
-To deploy the contracts first clone, install, and build the contracts package.
 
-Next set the following env vars:
+#### Required environment variables
+
+You must set the following environment variables to execute a deployment:
 
 ```bash
-CONTRACTS_TARGET_NETWORK=...
-CONTRACTS_DEPLOYER_KEY=...
-CONTRACTS_RPC_URL=...
-```
+# Name for the network to deploy to ("mainnet", "kovan", etc.)
```

Comment:
You have to create the genesis file first right?

Reply:
Oh yeah lemme include info about the genesis file